airflow_dags/dag.py

```diff
@@ -131,19 +132,37 @@ def get_args_helper(loader_builder: DoltLoaderBuilder, remote_url: str):
                                            bash_command='{{conf.get("core", "dags_folder")}}/neural_code_search_eval/import_from_source.pl ',
                                            dag=neural_code_search_eval_dag)
 
-# Wikipedia word frequency
-WIKIPEDIA_REPO = 'Liquidata/wikipedia-word-frequency'
+# Wikipedia dump variables
 DUMP_DATE = datetime.now() - timedelta(days=4)
-FORMATTED_DATE = DUMP_DATE.strftime("%-m-%-d-%y")
+FORMATTED_DATE = DUMP_DATE.strftime("%-m-%d-%y")
```

Comment:
`"%-m-%d-%y"` should "%m-%d-%y"  I think

Reply:
Changed to `20190801` instead of `8-01-2019`